### ISSUE TITLE
[kyc-match - address] no clear reference about what kind of address should be checked

### ISSUE THREAD

**Issue by GillesInnov35** [camaraproject/KnowYourCustomerMatch#42]
**Problem description**
There isn't any clear reference in the specifications about what kind of address should be checked on operator side
Should we require Home, address, Postal or Installation address ? 
- If not we let the operator define on which of these it'll base the check.
- We also know that it depends mostly on the back end solution called by the API.

**Possible evolution**
indicate clearly in API specifications what kind of address is required according to what information the user should be able to input.

BR
Gilles

**Comment by ToshiWakayama-KDDI** [comment]
Hi @PaoloCommissari , @HuubAppelboom , @GillesInnov35 , all,

Do we still need to keep this Issue open?

Thanks,
Toshi

**Comment by GillesInnov35** [comment]
hi @ToshiWakayama-KDDI, after an internal discussion, we wonder if we should not question the validity of this information returned regarding Data Minimization principles (GDPR requirements) ? 
The API will not return a match result but an information on contract. WDYT ?

**Comment by HuubAppelboom** [comment]
@ToshiWakayama-KDDI @GillesInnov35 @PaoloCommissari If privacy is a concern, mayeb we should only return the address match results for the when the addressType matches.  Otherwise you may be sharing accidentally information that is not supposed to be shared, not?    But what do you do then when the API Provider does not support this kind of classification ?

**Comment by GillesInnov35** [comment]
@HuubAppelboom , I agree with you. Indeed, there could be an issue in this scenario. This might mean that it is not interesting to include this property and to let the API provider apply its own rules by comparing with all known addresses or by limiting the scope to some of them.

**Comment by GillesInnov35** [comment]
@ToshiWakayama-KDDI , @HuubAppelboom , @PaoloCommissari as this interesting discussion is still in progress I propose to convert this issue to a discussion. WDYT ?
